### DISCUSSION TITLE
Does the Chat API retain context across multiple requests?

### DISCUSSION THREAD

**Discussion by koushikd9** [oobabooga/text-generation-webui#3109] [Q&A]
I am using the Chat API to generate responses to prompts. When I request a prompt, I get a response, but when I request a related prompt again, the API does not remember the context of the previous conversation. Is there a way to set a parameter so that the API will remember the context and generate more relevant responses

**Comment by Jewbby** [comment]
I've been trying to find a way around this too. Built out a STT to TTS with Ooba, Google, and Azure. Built all the parameter loading, character loading, user, and use complex model loading. However it seems no matter what there is a max of 250 tokens regardless of the limit being set both at 4096. After extensive testing, the conversation starts at 247-249, and each new response is just trimmed off the top, unable to add additional. I must be missing something, but after going through the API documentation that exists, I don't know..
